src/main/scala/com/codacy/client/bitbucket/service/BuildStatusServices.scala

```diff
@@ -0,0 +1,50 @@
+package com.codacy.client.bitbucket.service
+
+import com.codacy.client.bitbucket.BuildStatus
+import com.codacy.client.bitbucket.client.{BitbucketClient, Request, RequestResponse}
+import play.api.libs.json.Json
+
+class BuildStatusServices(client: BitbucketClient) {
+
+  /*
+   * Gets a commit build status
+   *
+   */
+  def getBuildStatus(owner: String, repository: String, commit: String, key: String): RequestResponse[BuildStatus] = {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Do not prefix getters with 'get'](https://www.codacy.com/app/Codacy/bitbucket-scala-client/file/1807008538/issues/source?bid=2474672&fileBranchId=3073963#l13)